validation/superchain-version_test.go

```diff
@@ -89,47 +64,24 @@ func TestContractVersions(t *testing.T) {
 		84532:     true, // sepolia/base
 		90001:     true, // sepolia/race, due to https://github.com/ethereum-optimism/superchain-registry/issues/147
 		7777777:   true, // mainnet/zora
+		11155420:  true, // sepolia/op
 		11155421:  true, // sepolia-dev-0/oplabs-devnet-0
+		11763072:  true, // sepolia-dev-0/base-devnet-0
 		999999999: true, // sepolia/zora
 	}
 
-	checkOPChainSatisfiesSemver := func(t *testing.T, chain *ChainConfig) {
-		// TODO: don't hardcode this https://github.com/ethereum-optimism/superchain-registry/issues/219
-		isFaultProofChain := map[uint64]bool{
-			10:       true, // OP Mainnet
-			11155420: true, // OP Sepolia
-		}
-
+	checkOPChainMatchesATag := func(t *testing.T, chain *ChainConfig) {
```

Comment:
For now, we should pass the check if the chain matches `1.3.0` or `1.4.0`. Eventually, when the standard config is ratified by Governance, it will need to be `1.4.0`.